### DISCUSSION TITLE
Idea 💡 Any plans to add pod view and management to pod manager?

### DISCUSSION THREAD

**Discussion by codeuh** [dreamcatcher45/podmanager#14] [Ideas]
H there dreamcatcher45! 

I hope this message finds you well. I very much enjoy your extension. I was curious if you had plans to implement a pod view into your extension. Potentially also the ability to create and delete pods? It only seems fitting with a name like pod manager 😁

I'm not well versed in typescript and VSCode extension development. It would take me a bit to spin up on that to be able to help contribute to your project.

Thanks,
codeuh

**Comment by dreamcatcher45** [comment]
@codeuh Thanks for the suggestion...In initial versions "pods" was a part of the extension, later it was merged with "compose-groups" section due to conflicts in the functionalities... Anyways I will try to bring back "pods" to the extension in future 😃👍

**Reply by codeuh** [reply]
Thank you for the quick response and additional insight. I'm not a compose user so I hadn't seen the "compose-groups" section yet. I appreciate the consideration of my suggestion.
